doc/source/whatsnew/v1.1.4.rst

```diff
@@ -22,7 +22,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Bug in :class:`BaseGroupedBy` causing ``groupby(...).sum()`` and similar to not preserve metadata (:issue:`29442`)
```

Comment:
you can remove the reference to BaseGroupedBy (this is not a real class) and these are user facing notes.